### DISCUSSION TITLE
plz help me. get_member()

### DISCUSSION THREAD

**Discussion by ins3821** [Rapptz/discord.py#6287] [Q&A] [answered]
```pytion
    if message.content.startswith("!test "):
        code = message.content[6:24]
        hi = message.guild.get_member(code)
        await message.channel.send(f"output:{hi} / Code:{code}")

```
that's what I wrote.
output 'output: None Code:123451234512345123'
this is the end result.
what's the problem?

**Comment by Harmon758** [comment, marked as answer]
Do you have the members privileged intent enabled and set?
https://discordpy.readthedocs.io/en/latest/intents.html

**Reply by ins3821** [reply]
thank you!!!!!

**Comment by 15696** [comment]
Please don't create duplicate questions. You already asked about this in #6285 so you should provide more details in there instead, than creating a whole new question.

**Reply by ins3821** [reply]
sorry, I got confused because it's my first time using it.
